deployment/7_upgrade_1_6.js

```diff
@@ -100,6 +101,20 @@ const deploy = async (network) => {
   );
   newModuleWrappers.push(MakerV2ManagerWrapper);
 
+  const TransferManagerWrapper = await deployer.deploy(
+    TransferManager,
+    {},
+    config.contracts.ModuleRegistry,
+    config.modules.TransferStorage,
+    config.modules.GuardianStorage,
+    config.contracts.TokenPriceProvider,
+    config.settings.securityPeriod || 0,
+    config.settings.securityWindow || 0,
+    config.settings.defaultLimit || "1000000000000000000",
+    ["test", "staging", "prod"].includes(network) ? config.modules.TokenTransfer : "0x0000000000000000000000000000000000000000",
```

Comment:
It should be `config.modules.TransferManager` and not `config.modules.TokenTransfer`.

Reply:
But `config.modules.TokenTransfer` is the address of the old `config.modules.TransferManager` no?

Reply:
Maybe when upgrading the TransferManager we should copy its old value here though

Reply:
It is the address of the first version of the TransferManager, yes. But here we are going from the second version (`config.modules.TransferManager`) to the third one. So the new TransferManager needs to read the state of a wallet's limit from the second version which is `config.modules.TransferManager`.

Reply:
But we update the `config.modules.TransferManager` address to the new one in the same script.

Reply:
Okay, removed `TokenTransfer` in favour of using `TransferManager` value prior to upgrade updates.